.github/ISSUE_TEMPLATE/request_change.yaml

```diff
@@ -0,0 +1,51 @@
+name: Feature request
+description: \U0001F680 Form to create a request for changes.
+title: "[FEATURE]: "
+labels: ["feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This form serves as a template to **create a request for changes**. It can be used, for example, to add a new section or functionality to the [FEGA-onboarding webspage](https://ega-archive.github.io/FEGA-onboarding/).
```

Comment:
```suggestion
        This form is a template to **create a request for changes**. It can be used, for example, to add a new section or functionality to the [FEGA onboarding webpages](https://ega-archive.github.io/FEGA-onboarding/).
```